modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -1,23 +1,34 @@
 // Module included in the following assemblies:
 //
-// * ingress/configure-ingress-operator.adoc
+// * networking/ingress-operator.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
 = Configuring the PROXY protocol for an Ingress Controller
 
-A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[the PROXY protocol] when an Ingress Controller uses either the `HostNetwork` or `NodePortService` endpoint publishing strategy types. The PROXY protocol enables the load balancer to preserve the original client addresses for connections that the Ingress Controller receives. The original client addresses are useful for logging, filtering, and injecting HTTP headers. In the default configuration, the connections that the Ingress Controller receives only contain the source address that is associated with the load balancer.
+A cluster administrator can configure link:https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[the PROXY protocol] when an Ingress Controller uses either the `HostNetwork` or `NodePortService` endpoint publishing strategy types.
 
-This feature is not supported in cloud deployments. This restriction is because when {product-title} runs in a cloud platform, and an IngressController specifies that a service load balancer should be used, the Ingress Operator configures the load balancer service and enables the PROXY protocol based on the platform requirement for preserving source addresses.
+[WARNING]
+====
+The PROXY protocol is unsupported for the default Ingress Controller with installer-provisioned clusters on non-cloud platforms that use a Keepalived Ingress VIP.
+====
+
+The PROXY protocol enables the load balancer to preserve the original client addresses for connections that the Ingress Controller receives. The original client addresses are useful for logging, filtering, and injecting HTTP headers. In the default configuration, the connections that the Ingress Controller receives only contain the source IP address that is associated with the load balancer.
 
 [IMPORTANT]
 ====
-You must configure both {product-title} and the external load balancer to either use the PROXY protocol or to use TCP.
+For a passthrough route configuration, servers in an {product-title} cluster cannot observe the original client source IP address. If you need to know the original client source IP address, ensure that you configure Ingress access logging for your Ingress Controller, so that you can view the client source IP addresses.
```

Comment:
```suggestion
For a passthrough route configuration, servers in an {product-title} cluster cannot observe the original client source IP address. If you need to know the original client source IP address, configure Ingress access logging for your Ingress Controller so that you can view the client source IP addresses.
```

Reply:
:heavy_check_mark: